CHANGELOG/CHANGELOG-0.5.md

```diff
@@ -0,0 +1,28 @@
+# Changelog 0.5
+
+## 0.5.0 - 2023-03-17
+
+### Added
+
+- Added Theia Manager resync at startup. ([#150](https://github.com/antrea-io/theia/pull/150), [@yanjunz97])
+  * Support removing stale Spark jobs and database entries after restarting Theia Manager.
+  * Scheduled Network Policy Recommendations jobs are added back to the periodical sync list.
+- Service Dependency Graph. ([#142](https://github.com/antrea-io/theia/pull/142), [@Dhruv-J])
+  * Introduced a Sankey diagram for visualizing traffic between Pods and Services.
```

Comment:
Is it a Sankey diagram? Maybe should we consider it a "directed graph"?

Reply:
changed to directed graph